index.html

```diff
@@ -1061,8 +1061,47 @@ <h3>Proof Chains</h3>
         </pre>
 
       </section>
+
+      <section>
+        <h3>Proof Graphs</h3>
+
+        <p>
+When securing data in a document, it is important to clearly delineate the data
+being protected, called the <dfn class="export">default graph</dfn>, from the
+data associated with a securing mechanism, called a
+<dfn class="export">proof graph</dfn>. Creating this separation enables the
+processing algorithms to be able to deterministically protect and verify a
+secured document.
+        </p>
+
+        <p>
+For an unsecured document, the information contained in the document before a
+[=data integrity proof=] is added to the document is called the [=default
+graph=]. A single document can contain multiple proof graphs. To ensure that
+information from different [=data integrity proofs=] are not accidentally
+co-mingled, the concept of a [=proof graph=] is used to encapsulate each [=data
+integrity proof=]. Each value associated with the `proof` property of the
+document is a separate graph, which is sometimes referred to as a
+<dfn class="export">named graph</dfn>, of type
+<dfn class="export">ProofGraph</dfn>, which contains a single [=data integrity
+proof=].
```

Comment:
Also aimed at addressing Ivan's concerns:

```suggestion
For an unsecured document, the information contained in the document before a
[=data integrity proof=] is added to the document is expressed in one or more
graphs. A single document can also contain multiple proof graphs. To ensure that
information from different [=data integrity proofs=] are not accidentally
co-mingled, the concept of a [=proof graph=] is used to encapsulate each [=data
integrity proof=]. Each value associated with the `proof` property of the
document is a separate graph, which is sometimes referred to as a
<dfn class="export">named graph</dfn>, of type
<dfn class="export">ProofGraph</dfn>, which contains a single [=data integrity
proof=].
```